tests/python/contrib/test_hexagon/metaschedule_e2e/test_resnet50_int8.py

```diff
@@ -208,7 +215,7 @@ def schedule_fn(sch, conv2d_block: Optional[BlockRV] = None) -> bool:
         if conv2d_block is None:
             try:
                 conv2d_block = sch.get_block("conv2d_NCHWc_int8")
-            except ValueError:
+            except:
                 return False
```

Comment:
I added `has_block` recently here https://github.com/apache/tvm/blob/f42826eec49998452cff30a2b6510e7d3c31e3ec/python/tvm/tir/schedule/analysis.py#L127. It can be used to remove this try / catch.

Reply:
Cool new feature! Added ✅